### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Fostrankill**
Updated search buton and carousel

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -14,10 +13,7 @@ function App() {
           <Logo />
           <Titre />
         </div>
-        <div className="caroussel">
-          <Card />
-          <CardBack />
-        </div>
+        <Slideshow />
         <div id="searchbtn">
           <SearchButton />
```
Comment (Fbouchezpro) [inline frontend/src/App.jsx:20]: slideshow in div classname=caroussel

**Review by Fbouchezpro — CHANGES_REQUESTED** [review]

**Review by Fbouchezpro — APPROVED** [review]

File: frontend/package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
+    "react-card-carousel": "^1.1.3",
```
Comment (LoicBrassart) [inline frontend/package.json:15]: Poopy lib is poopy

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,9 +1,8 @@
-import Card from "@components/Cards/Card/Index";
-import CardBack from "@components/Cards/CardsBack/Index";
+import Slideshow from "./components/Carrousel/Caroussel";
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:1]: Choose a spelling and stick to it

File: frontend/src/components/Cards/Card/Index.jsx [hunk 4]
```diff
@@ -1,9 +1,11 @@
+import "../../../services/reset.css";
```
Comment (LoicBrassart) [inline frontend/src/components/Cards/Card/Index.jsx:1]: Already told you: stop importing this everywhere

File: frontend/src/components/Cards/Card/Index.jsx [hunk 5]
```diff
@@ -1,9 +1,11 @@
...
+import PropTypes from "prop-types";
 import "./style.css";
 
-export default function Card() {
+export default function Card({ hoverable }) {
   return (
     <section className="frontImage">
-      <div className="card">
+      <div className={`card ${hoverable ? "hoverable" : ""}`}>
```
Comment (LoicBrassart) [inline frontend/src/components/Cards/Card/Index.jsx:8]: You told in propTypes that `hoverable` is required -> no need to test it's existence

File: frontend/src/components/Carrousel/Caroussel.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useState } from "react";
+import ReactCardCarousel from "react-card-carousel";
+import Card from "../Cards/Card/Index";
+
+function Slideshow() {
+  let carousel = useRef(null);
```
Comment (LoicBrassart) [inline frontend/src/components/Carrousel/Caroussel.jsx:6]: Explain useRef to me =D

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils to fix !

File: frontend/src/components/Carrousel/Caroussel.jsx (hunk 6 above)
Comment (Fbouchezpro) [reply frontend/src/components/Carrousel/Caroussel.jsx:6]: A hook no? :-D

**Pull request closed without merging**